pkg/kube/fake_pod_command_executor.go

```diff
@@ -0,0 +1,28 @@
+package kube
+
+import (
+	"context"
+	"io"
+)
+
+type FakeKubePodCommandExecutor struct {
```

Comment:
Same suggestion about remove the word `Kube`

Reply:
done